Pint 40 add docstrings (#6)
Iteo/audoma: docs/build/html/_sources/usage.rst.txt

| Otherwise this will be displayed as default choices, so in API, as value there will be the databse value.

Filters
=======

Default Filters
----------------

| In drf it's possible to define `filterset_fields` and `filterset_class`.
| By default drf-spectacular supports django-filters. Which are being documented.
| Audoma has been tested with default drf's filter backend and `DjangoFilterBackend`.
| For more accurate documentation we recommend to use `DjangoFilterBackend` as default one.
| Filters and search fields are being documented out of the box automatically:

.. code :: python

    class CarViewSet(
    mixins.ActionModelMixin,
    mixins.RetrieveModelMixin,
    mixins.ListModelMixin,
    viewsets.GenericViewSet,
    ):
        queryset = Car.objects.none()
        serializer_class = CarModelSerializer

        filter_backends = [SearchFilter, df_filters.DjangoFilterBackend]

        filterset_fields = ["engine_type"]
        search_fields = ["=manufacturer", "name"]

| It is also possible to define filterset class which will also be documented
| without any additional steps.

| The main extension of this feature in audoma is additional enum documentation.

> | In `drf-spectacular` enums are being shown only as values possible to pass to the filter.

Author: ```suggestion
| In `drf-spectacular`,  enums are being shown only as values possible to pass to the filter.
```
comma 😺

> | Using audoma you also get a display field of a choice, this may be useful to show display value

Author: ```suggestion
| With audoma, you also get a display field of a choice, this can be useful to show display value
```
